src/Components/People.js

```diff
@@ -0,0 +1,75 @@
+import { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  state = { input: "", people: [], hasSearched: false, ids: {} };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    this.setState({ hasSearched: true });
+    try {
+      const { input } = this.state;
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/people/`);
+      this.setState({ people: res.data });
+      
+    } catch (err) {
+      this.setState({ people: {} });
+    }
+    this.setState({ input: "" });
+    debugger;
+    {
+      const names = this.state.people.map((movie) => {
+        const movieContainer = {};
+        movieContainer[movie.name] = movie.id;
+        return movieContainer;
+      });
+    //   console.log(names);
+      this.setState({ ids: names })
+    }
+    // console.log(this.state)
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+  };
+  render() {
+    const { input, people, hasSearched} = this.state;
+
+    let text; 
+    if(hasSearched) {
+        if(people.name) {
```

Comment:
people in an array, so you cannot key into it. Therefore you will always fail this conditional which is why you always show Person Not Found! so it fails the final test by default